### PR TITLE
This adds 2 new functions to add CSS variable support

### PR DESCRIPTION
**Pull request by GrimLink**
This MR also includes a small update to the readme, to make it easier to view what is available.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,50 +67,137 @@ const colors = require(`${themeDirRequire}/tailwindcss/colors`);
...
+and with this you can change the value in your CSS or inline in the page with:
+
+```css
+:root {
+    --color-primary: hsl(20 80% 50%);
+}
+```
+
+Now if you don't want to set this for each TailwindCSS token we also offer the functions `twProps`.
+This acts as a wrapper and will use the keys as the name for the CSS variable,
```
Comment (Vinai) [inline README.md:149]: Please also add a shorter example that clarifies how the variable names are built, maybe with just one or two lines.

Keep the longer example you used below to showcase how it looks in more context.

File: README.md [hunk 2]
```diff
@@ -58,50 +67,137 @@ const colors = require(`${themeDirRequire}/tailwindcss/colors`);
...
+    }
+    // The rest of your tailwind config...
+})
+```
+
+**Important to note:**
+
+- `twVar` only works with string values.
+- `twProps` can be used on any level of your Tailwind config, but it only works with string values.
+- `twProps` is best used inside the [reference function method](https://tailwindcss.com/docs/theme#referencing-other-values).
```
Comment (Vinai) [inline README.md:182]: This probably needs an example.

File: README.md [hunk 3]
```diff
@@ -58,50 +67,137 @@ const colors = require(`${themeDirRequire}/tailwindcss/colors`);
...
+- You can use `Object.assign()` to split 2 groups inside any Tailwind config group, e.g. `colors`, so only one part gets the variables and the other part is left as is.
```
Comment (Vinai) [inline README.md:183]: This also needs an example.

File: README.md [hunk 4]
```diff
@@ -58,50 +67,137 @@ const colors = require(`${themeDirRequire}/tailwindcss/colors`);
...
+- Both `twVar` and `twProps` have a prefix argument, this also highlighted when using Intellisense in your editor, powered trough JSDoc.
```
Comment (Vinai) [inline README.md:184]: This also needs an example. The " powered trough JSDoc" can be removed as it is an implementation detail.

File: src/index.js [hunk 5]
```diff
@@ -8,6 +8,7 @@ const deepmerge = require('deepmerge')
 const fs = require('fs')
 const path = require('path');
 const {cwd} = require('process');
+const { twVar, twProps } = require('./tailwind-css-props');
```
Comment (Vinai) [inline src/index.js:11]: Please also add spaces around `cwd` on line 10, or remove the spaces on line 11 :)

File: src/index.js [hunk 6]
```diff
@@ -195,5 +196,9 @@ module.exports = {
   // Parsed contents of the app/etc/hyva-themes.json file (or false)
   hyvaThemesConfig,
   // Array of absolute paths to Hyvä modules
-  hyvaModuleDirs
-}
+  hyvaModuleDirs,
+  // Function to add the new CSS variable syntax from Tailwind v4
```
Comment (Vinai) [inline src/index.js:200]: Please remove `new`.

File: src/tailwind-css-props.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
+/**
+ * Hyvä Themes - https://hyva.io
+ * Copyright © Hyvä Themes 2022-present. All rights reserved.
+ * This library is distributed under the BSD-3-Clause license.
+ */
+
+/**
+ * Check if string looks like a color
+ * This does not valid date if the color value is a valid color and does not support named colors
```
Comment (Vinai) [inline src/tailwind-css-props.js:9]: Typo: "valid date" -> "validate"

**Review by Vinai — CHANGES_REQUESTED** [review]
Excellent work, thank you! Just some minor changes required in regards to the text.

File: src/index.js (hunk 5 above)
Comment (GrimLink) [reply src/index.js:11]: Me avoiding any extra formating 😅, but consider it done.

**Review by Vinai — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**